### ISSUE TITLE
[Feature request] Add an option BatchNorm => Instance norm

### ISSUE THREAD

**Issue by ternaus** [qubvel/segmentation_models.pytorch#294]
I would like to be able to replace BatchNorm with Instance norm in the network at the initialization, including pre-trained backbones.

**Comment by ternaus** [comment]
Actually, this could be done with

```
def bn2instance(module):
    module_output = module
    if isinstance(module, torch.nn.modules.batchnorm._BatchNorm):
        module_output = torch.nn.InstanceNorm2d(module.num_features,
                                                module.eps, module.momentum,
                                                module.affine,
                                                module.track_running_stats)
        if module.affine:
            with torch.no_grad():
                module_output.weight = module.weight
                module_output.bias = module.bias
        module_output.running_mean = module.running_mean
        module_output.running_var = module.running_var
        module_output.num_batches_tracked = module.num_batches_tracked
        if hasattr(module, "qconfig"):
            module_output.qconfig = module.qconfig

    for name, child in module.named_children():
        module_output.add_module(name, bn2instance(child))

    del module
    return module_output
```
